src/libraries/System.Private.CoreLib/src/System/IndexOfAnyValues/IndexOfAnyAsciiSearcher.cs

```diff
@@ -809,6 +809,9 @@ internal static int LastIndexOfAnyVectorized<TNegator>(ref byte searchSpace, int
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [System.Runtime.BypassReadyToRunForIntrinsicsHelperUse(typeof(Sse))]
+        [System.Runtime.BypassReadyToRunForIntrinsicsHelperUse(typeof(AdvSimd))]
+        [System.Runtime.BypassReadyToRunForIntrinsicsHelperUse(typeof(PackedSimd))]
```

Comment:
Nit:
```suggestion
        [BypassReadyToRunForIntrinsicsHelperUse(typeof(Sse))]
        [BypassReadyToRunForIntrinsicsHelperUse(typeof(AdvSimd))]
        [BypassReadyToRunForIntrinsicsHelperUse(typeof(PackedSimd))]
```

Should it be `Sse2` instead of `Sse` here?

Reply:
Yes. This is actually showing one of the limitations of the analyzer, which is that if you follow the pattern of having a helper which uses multiple variants which are explicitly checked, it isn't good at requiring the exact right attribute.

Reply:
I should be able to make the analyzer detect this situation and issue a slightly different diagnostic, which could be suppressed with a pragma warning disable for legit cases.